operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -669,6 +671,23 @@ private void internalMakeRightDomainPresence() {
             willInterrupt);
     }
 
+    private void populateServerStateHealthFromDomain(Packet packet) {
+      Map<String, ServerHealth> serverHealth = new HashMap<String, ServerHealth>();
+      Map<String, String> serverState = new HashMap<String, String>();
```

Comment:
Should these be ConcurrentHashMap instead of regular HashMap?

Reply:
yes, good catch! Changed to ConcurrentHashMap.

Reply:
The scheduleDomainStatusUpdating creates a new packet for periodically domain status update, which is not the same packet that operator creates here in internalMakeRightDomainPresence.